int32 First = -1 
int32 Last = -2
int32 Current = -3
int32 Standard = -2
string WaitCommand = "#CWait"

string destination
int32 indexInQueue
int32 moveTheta
---
int32 orderID